src/main/webapp/app/exercises/programming/participate/code-editor-student-container.component.html

```diff
@@ -1,88 +1,91 @@
-<nav *ngIf="loadingParticipation || participationCouldNotBeFetched" class="card">
-    <div class="card-body">
-        <span *ngIf="loadingParticipation">
-            <fa-icon [icon]="faCircleNotch" [spin]="true"></fa-icon>
-            <span jhiTranslate="artemisApp.editor.fetchingParticipation"> Your participation is being loaded... </span>
-        </span>
-        <span *ngIf="participationCouldNotBeFetched">
-            <fa-icon [icon]="faTimesCircle" class="text-danger"></fa-icon>
-            <span jhiTranslate="artemisApp.editor.errors.participationNotFound">Your participation could not be found.</span>
-        </span>
+<div [id]="'exercise-' + exercise?.id">
+    <nav *ngIf="loadingParticipation || participationCouldNotBeFetched" class="card">
+        <div class="card-body">
+            <span *ngIf="loadingParticipation">
+                <fa-icon [icon]="faCircleNotch" [spin]="true"></fa-icon>
+                <span jhiTranslate="artemisApp.editor.fetchingParticipation"> Your participation is being loaded... </span>
```

Comment:
IMO paragraph
```suggestion
                <p jhiTranslate="artemisApp.editor.fetchingParticipation"> Your participation is being loaded... </p>
```

Reply:
although [docs](https://docs.artemis.cit.tum.de/dev/guidelines/client/#icons-with-text) suggests spans. I still think paragraphs are better for accessibility reasons

Reply:
See comment above